src/test/java/com/ibm/cloud/sdk/core/test/service/ErrorResponseTest.java

```diff
@@ -124,18 +174,18 @@ public void testUnauthorized() {
   public void testForbidden() {
 
     String message = "The request failed because the moon is full.";
+    String responseBody = "{\"errors\": [{\"message\": \"" + message + "\"}, {\"message\": \"another error...\"}]}";
```

Comment:
We didn't have a test that exercises the "errors" field (which is supposed to be a list).